qiskit/ignis/verification/tomography/__init__.py

```diff
@@ -13,8 +13,68 @@
 # that they have been altered from the originals.
 
 
-"""
-Quantum State and Process Tomography module
+"""Quantum State and Process Tomography module
```

Comment:
This is a problem with the docs/ignis.rst part of the project; I've attempted to modify it but it creates conflicts I prefer to avoid for now. My suggestion is that after the all the Ignis docs are merged we'll have a new PR to directly address the TOC structure in the docs for all the Ignis components.